HelpfulItem.h

```diff
@@ -10,6 +10,6 @@
 struct HelpfulItem : Item
```

Comment:
Don't **include** what you don't use!  A bunch of files are being included above.
- [ ] revise